frontend/src/navbar.tsx

```diff
@@ -86,6 +99,10 @@ function isPullCryo(pull: Pull): boolean {
    return !pull.getLabel("Cryogenic Storage");
 }
 
+function isPullExtBlocked(pull: Pull): boolean {
+   return !pull.getLabel("external_block");
+}
```

Comment:
This logic seems backwards to me... shouldn't this function return true if the `external_block` label is on the PR? Aren't we currently saying "if the pull doesn't have the `external_block` label, return true?"

Reply:
**getLabel** returns the label string using find(), if whatever passed as a parameter is present on a specific pull

     * Returns the value of the first element in the array where predicate is true, and undefined
     * otherwise.

The `!` here will make this function return a boolean value based on it's return value

Reply:
Derp, thanks for the info!

Reply:
https://github.com/iFixit/pulldasher/commit/35ba36c68c1a3b2f134f477307ea570a3e1010d2

Reply:
I think it's just function name that's wrong. If the label is present, `getLabel` will return something truthy and this function will negate that and return false. I think it should be named `isNotExternallyBlocked`. The same goes for `isPullCryo` for that matter.

Reply:
Resolved via https://github.com/iFixit/pulldasher/pull/334/commits/f373bca3fabe86ffc85afaea2dd68f460a1bb408